Kinvey.Tests/BaseTestClass.cs

```diff
@@ -81,15 +81,15 @@ public static string AppSecret
         [TestInitialize]
         public virtual void Setup()
         {
-            System.IO.File.Delete(TestSetup.SQLiteOfflineStoreFilePath);
-            System.IO.File.Delete(TestSetup.SQLiteCredentialStoreFilePath);
+            //System.IO.File.Delete(TestSetup.SQLiteOfflineStoreFilePath);
```

Comment:
feel free to delete those lines, you don't have to comment them. If we need them back we can always look the git history.

Reply:
@heyzooi  Thank you! 
I agree with.